lib/cloud_controller/opi/apps_client.rb

```diff
@@ -104,10 +104,35 @@ def to_hash
       end
     end
 
+    class KpackLifecycle
+      CNB_LAUNCHER_PATH = '/cnb/lifecycle/launcher'.freeze
+
+      def initialize(process)
+        @process = process
+      end
+
+      def to_hash
+        command = if @process.started_command.presence
+                    ['/bin/sh', '-c', "#{CNB_LAUNCHER_PATH} #{@process.started_command}"]
```

Comment:
🤔  is the `bin/sh` necessary here? it's safe to leave it in but it's kinda interesting to think about.

Reply:
Yeah, I thought that as well. I went with this cause it's what we were doing for docker droplets, but 🤷 .
https://github.com/cloudfoundry/cloud_controller_ng/pull/1417/files#diff-738bfbb749a5827ff834ca07b3594487e3f321066cf64d20ff4bdf7359137f4bR74

I can ask the Eirini folks.